homework_03/02_Angel_Stoyanov/RelativePosition.py

```diff
@@ -0,0 +1,7 @@
+import enum
+
+class RelativePosition(enum.Enum):
+    NO_COMMON_POINTS = 1
+    TOUCHING = 2
+    INTERSECTING = 3
+    SAME = 4
```

Comment:
I like that you have spread your code in multiple files, that way it is more organized!